### PR TITLE
Accessible buttons

### PR DESCRIPTION
**Pull request by arthur801031**
#1269

### CODE REVIEW

**Comment by jasonLaster** [comment]
Thanks! Looking good.

File: src/components/Dropdown.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ const Dropdown = React.createClass({
   },
 
   renderButton() {
-    return dom.span(
+    return dom.button(
```
Comment (jasonLaster) [inline src/components/Dropdown.js]: This adds some funny styles

You should be able to reproduce by adding some more editor tabs an observing the hidden tabs dropdown appear.

screen shot 2016-11-26 at 8 27 05 pm

**Comment by jasonLaster** [comment]
screen shot 2016-11-27 at 10 15 10 am

noticed chrome got some useragent styling that was easy to clean up with some background/border noneing

**Comment by arthur801031** [comment]
@jasonLaster  Thank you very much for your help! I really appreciate it! :)

**Pull request merged**